docs/managers.rst

```diff
@@ -70,6 +70,20 @@ connections and individual per-server:port
 :class:`~urllib3.connectionpool.HTTPSConnectionPool` instances for tunnelled
 HTTPS connections.
 
+Example using proxy authentication:
+
+::
+
+	>>> import base64
```

Comment:
Don't worry about the "connection reset by peer" errors, that's just travisci machines being overloaded. :(